### ISSUE TITLE
cross-compilation error with v7.0.4

### ISSUE THREAD

**Issue by vrothberg** [vbauerster/mpb#94]
v7.0.4 seems to have a build issue on windows.  The build failed in the CI of github.com/containers/image (https://github.com/containers/image/pull/1329/checks?check_run_id=3241064046):

```
[...]
GOOS=windows make build BUILDTAGS="btrfs_noversion libdm_no_deferred_remove containers_image_openpgp"
[...]
CGO_CFLAGS="" CGO_LDFLAGS="-L/usr/lib/x86_64-linux-gnu -lgpgme -lassuan -lgpg-error" GO111MODULE="on" go build -tags "btrfs_noversion libdm_no_deferred_remove containers_image_openpgp" ./...
# github.com/vbauerster/mpb/v7/cwriter
/var/tmp/go/pkg/mod/github.com/vbauerster/mpb/v7@v7.0.4/cwriter/writer_windows.go:29:34: w.lineCount undefined (type *Writer has no field or method lineCount)
/var/tmp/go/pkg/mod/github.com/vbauerster/mpb/v7@v7.0.4/cwriter/writer_windows.go:43:41: w.lineCount undefined (type *Writer has no field or method lineCount)
```

**Comment by vbauerster** [comment]
Yes my bad, forgot about windows. Fixed in v7.0.5.

**Comment by vrothberg** [comment]
That was quick, thank you, @vbauerster
